README.md

```diff
@@ -28,6 +28,45 @@ pip install git+https://github.com/huggingface/optimum-intel.git
 
 # Quick tour
 
+## Neural Compressor
+
+#### Dynamic quantization:
+
+Here is an example on how to apply dynamic quantization on a DistilBERT fine-tuned on the SQuAD1.0 dataset.
+Note that quantization is currently only supported for CPUs (only CPU backends are available), so we will not be utilizing GPUs / CUDA in this example.
+
+```python
+from transformers import AutoModelForQuestionAnswering
+from neural_compressor.config import PostTrainingQuantConfig
+from optimum.intel.neural_compressor import INCQuantizer
+
+model_name = "distilbert-base-cased-distilled-squad"
+model = AutoModelForQuestionAnswering.from_pretrained(model_name)
+# The directory where the quantized model will be saved
+save_dir = "quantized_model"
+# Load the quantization configuration detailing the quantization we wish to apply
+quantization_config = PostTrainingQuantConfig(approach="dynamic")
+quantizer = INCQuantizer.from_pretrained(model)
```

Comment:
Ella, do you have any future plan to have a AutoQuantizer liker Transformers to automatically dispatch to right quantizer?

Reply:
What do you mean by an `AutoQuantizer` dispatching to the right quantizer ?